### ISSUE TITLE
Ideas for mobile development

### ISSUE THREAD

**Issue by hng-teamfury** [hng-teamfury/stage2#7]

**Comment by fridaychukwuamaka** [comment]
Pls, we should vote on the mobile development tool to use

**Comment by bayodekehinde** [comment]
Since flutter is for cross platform...I think we should use it

**Comment by fridaychukwuamaka** [comment]
I also think so, but we need a full house to be able to decide

**Comment by maroafenogho** [comment]
Good evening guys.
It took me a little time to join in. 
I believe using flutter would be great for everyone. It would also help people learn how the flutter environment works

**Comment by maroafenogho** [comment]
We also need to come up with a plan on which app to build for this task. 
This needs to be done before 9:00 AM tomorrow.
We have had some suggestions <a href="https://hngi7.slack.com/archives/G0155KA37M2/p1591560036079000">here</a>

**Comment by acefestus** [comment]
Great ideas shared already on the Slack thread and here already. Please, as we brainstorm, we should consider how this app can be easily be used for another purpose without redesigning as Mark has stated. Also, when we decided on the app we are making, the research team can help find out if such an app already exists, and we move on from there.

**Comment by Francisop** [comment]
covid 19 prevention app was suggested earlier as an app that reminds users to wash their hands and it can be easily modified ...

**Comment by fridaychukwuamaka** [comment]
Is the issue of flutter being the main development tool been settled

**Comment by fridaychukwuamaka** [comment]
If we are using Flutter, we need to know those who are good or have an in flutter

**Comment by maroafenogho** [comment]
If by 8 tomorrow we do not have enough reason not to use flutter, we will be using flutter

**Comment by acefestus** [comment]
I am sure we have people who are good with Flutter here already, before we talk about what platform to use, let us solidify the idea.

**Comment by maroafenogho** [comment]
I think most of these people have gone to sleep. We need to come up with something and fast.

**Comment by maroafenogho** [comment]
We need to have a solid idea to present to Mark during the 12 o'clock call tomorrow.

**Comment by acefestus** [comment]
yeah

Can we continue with anyone online at the moment?

**Comment by Francisop** [comment]
i vote the covid 19 prevention app

**Comment by maroafenogho** [comment]
Sure

**Comment by maroafenogho** [comment]
Let's bring up ideas

**Comment by maroafenogho** [comment]
I think a covid app would be great also.

**Comment by Francisop** [comment]
@maroafenogho seconded

**Comment by acefestus** [comment]
Who is still here?

I discussed with the frontend people, they will help get the design.

Let us agreed on the covid app, what and what will it do?
the general team lead shared this with me for our track
https://github.com/hng-teamfury-org/mobile

**Comment by johnsani** [comment]
Convid 19 app is a good idea. But what other five ways can the app be used?

**Comment by bayodekehinde** [comment]
Any app we decide to develop must have a common feature that can let the app be easily modified to different app. Since covid19 is a disease/infection  prevention related app, we can use for any other disease

**Comment by maroafenogho** [comment]
Good morning guys. The suggestion on slack was a hand wash reminder. Now that can easily be modified to a drink water reminder, a reading reminder, a pill reminder and so much more.

**Comment by maroafenogho** [comment]
We need to agree on this by 9am

**Comment by johnsani** [comment]
> We need to agree on this by 9am

Exactly. I just dropped more ideas on slack. You guys can check it out. But if we're going with the hand washing app, then I suggest we start working on it to be a step ahead.

**Comment by johnsani** [comment]
Please who is the lead for the mobile team?

**Comment by Trojan9** [comment]
I am the team lead

**Comment by Trojan9** [comment]
I think we should go with the covid19 prevention reminder... The one we talked about on slack

**Comment by Trojan9** [comment]
Since it can be easily modified

**Comment by acefestus** [comment]
Hello guys, good morning. Three of us did a little brainstorming before 2 am this morning. I tried to edit the README of the mobile track on Github but I do not have the right to do so.

This is the summary of the discussion.  However, I think we should stay in one place for discussion. Discussion cannot go at the same time in two places. Either we stay here or stay on slack.


The following were discussed before 2am today. However, other members of the team are given the priviledge to share more idea latest by 9am today (8th June 2020) to make the idea better before we move forward.
## Idea -
Develop a hand-washing reminder app for Covid19 prevention
Primary feature of the app
  *remind (alarm with a certain message) to wash their hands
Secondary features
  *Set up various tasks for Covid19 prevention e.g. time to c
## Tasks
1. We ask the designer/frontend team to make a design, using Adobe XD, because flutter is able to convert Adobe XD designs to code
## Design
1. The app will first be designed with the primary features
2. Secondary features can be added later

**Comment by Dera-O** [comment]
> Hello guys, good morning. Three of us did a little brainstorming before 2 am this morning. I tried to edit the README of the mobile track on Github but I do not have the right to do so.
> 
> This is the summary of the discussion. However, I think we should stay in one place for discussion. Discussion cannot go at the same time in two places. Either we stay here or stay on slack.
> 
> The following were discussed before 2am today. However, other members of the team are given the priviledge to share more idea latest by 9am today (8th June 2020) to make the idea better before we move forward.
> 
> ## Idea -
> Develop a hand-washing reminder app for Covid19 prevention
> Primary feature of the app
> *remind (alarm with a certain message) to wash their hands
> Secondary features
> *Set up various tasks for Covid19 prevention e.g. time to c
> 
> ## Tasks
> 1. We ask the designer/frontend team to make a design, using Adobe XD, because flutter is able to convert Adobe XD designs to code
> 
> ## Design
> 1. The app will first be designed with the primary features
> 2. Secondary features can be added later

Sounds good to me. Has it been made known to designers yet?

**Comment by maroafenogho** [comment]
I have an idea for the COVID-19 prevention app. @damisco87 it was stated that discussions be done here. It helps us keep track of what we have discussed.

**Comment by acefestus** [comment]
@Trojan9 please moderate the discussion, let us move ahead.

**Comment by maroafenogho** [comment]
Since we have all agreed on the COVID-19 prevention app, I have an idea on it.

**Comment by Trojan9** [comment]
okay now that we have agreed on the prevention app

**Comment by Trojan9** [comment]
let ideas on how to improve it start coming in

**Comment by Trojan9** [comment]
first we need a UI design for that

**Comment by maroafenogho** [comment]
@damisco87 in addition to what you have said, I think it can be more than just hand washing. 
Maybe the reminder could have an alert like:"Did you just touch any surface? Wash your hand. Can't wash at the moment, no problem. Sanitize"

**Comment by acefestus** [comment]
I already discussed with one of the frontend guys last night, they will help design it

**Comment by Dera-O** [comment]
People should be able to keep track of their handwashing activity. Showing and giving kudos for progress has a way of keeping people engaged in something

**Comment by Trojan9** [comment]
okay..do you think we can get the design today...so we can beat the timeframe

**Comment by acefestus** [comment]
Let me talk to them now. 

@Trojan9 I learnt Mark wants us to submit our app idea this morning. If so, you would need to communicate with the team lead

**Comment by Trojan9** [comment]
so we have two additional modules already...please lets not forget to keep it simple

**Comment by vectaaa** [comment]
Hello guys good day....the covid19 app isn't going to so useful in a few months time...lets make the face of the app either be a hydration timer for athletes, a timer for basic food recipes, or something else...not covid19 pls. There are plenty apps like that online already...we still have 10mins tho

**Comment by maroafenogho** [comment]
@vectaaa the app is for the this task. That's why it was stated, "something that can easily be modified"
It's using the current situation of things to our advantage.

**Comment by maroafenogho** [comment]
@Trojan9 what do you mean by two additional modules please?

**Comment by vectaaa** [comment]
I also have one, its called an appointment scheduler it works like a todo app...and it will enable professionals schedule appointments they have for the week during the weekend and even send mails through the apps for canceled or rescheduled appointments

**Comment by acefestus** [comment]
@maroafenogho and @vectaa, thanks for your ideas. Vectaa, when covid is gone, we can easily change it to anything we want.

**Comment by acefestus** [comment]
Should I go ahead and discuss with the frontend people for the design?

**Comment by Trojan9** [comment]
@maroafenogho  i am dividing the app into different modules....so we have people working on different modules to fasten the work

**Comment by Trojan9** [comment]
@damisco87 yes please do

**Comment by vectaaa** [comment]
@damisco87 remember we are going to do marketing and in marketing what you put out as your product stays in the customers minds...
Honestly lets be straight here who has a covid19 help app on his phone and uses it?

I bet you a very few people......
Well i would work any app chosen oh...i just needed to say my mind tho

**Comment by Trojan9** [comment]
@damisco87 is making sense though

**Comment by maroafenogho** [comment]
@vectaaa I understand what you're saying. I also want you to consider the task given. The app must be easily modified into something else. The simpler the better. It can be easily modified if there is a need to. 
@Trojan9 have you notified the team lead on the app idea?

**Comment by acefestus** [comment]
Hello all, kindly DM me your information below

slack name | github name | track

# example
@piouson backend
@simply_alliv | @allistairvilakazi | frontend

**Comment by acefestus** [comment]
I have informed the Frontend guys, they will get back to me.

**Comment by fridaychukwuamaka** [comment]
Pls, we should not be too forward, we should just set things down and await for the Idea to be approved by Mark before going forward

**Comment by acefestus** [comment]
> Pls, we should not be too forward, we should just set things down and await for the Idea to be approved by Mark

Oh. is that so?

**Comment by robincruseo** [comment]
So far, so good, has any of the projects been selected yet, because i do wish and hope to get involved by the way my slack /github/ track details respectively is found below
slack: @robincruseo
github: @robincrueo
track: mobile

**Comment by Trojan9** [comment]
@TimmyBanjo|| @Trojan9  || mobile

**Comment by payn33** [comment]
Hey guys, frontend here don't know much about mobile dev but I have an idea if it's not too late. A simple workout reps and sets counter and timer app, could also be repurposed as a running laps counter and timer, or even a recipe app and food counter. What do y'all think?

**Comment by judexes** [comment]
@payn33 honestly I prefer your idea to the covid 19 app in the long run, but thing is the covid 19 app will be interesting for users to download and play around with as they would want to know if it works and how it works because that's currently what is trending

**Comment by fridaychukwuamaka** [comment]
@vectaaa the covid 19 app can be used after covid 19, because people may continue with the habit of washing hands regularly

**Comment by judexes** [comment]
My slack username is: judeokoroafor

**Comment by johnsani** [comment]
I prefer the convid 19 hand washing idea. We have a better chance of convincing people to download the app. My slack username is @nimi

**Comment by acefestus** [comment]
> Hello all, kindly DM me your information below
> 
> slack name | github name | track
> 
> # example
> @piouson backend
> @simply_alliv | @allistairvilakazi | frontend

Please note, I will only compile the names of the active participants here.

**Comment by acefestus** [comment]
Also, the #team-furry page on slack, volunteers were called for to documents discussion and for giving regular update. Do we have someone interested to do this for the team?
https://hngi7.slack.com/archives/G0155KA37M2/p1591570646363100?thread_ts=1591570646.363100&cid=G0155KA37M2

**Comment by fridaychukwuamaka** [comment]
Please have decided on the idea we are submit before 10am

**Comment by acefestus** [comment]
> Please have decided on the idea we are submit before 10am

Yes, covid app for now, we can switch the app later

**Comment by acefestus** [comment]
Where is @Trojan9 ?

**Comment by Dera-O** [comment]
> Also, the #team-furry page on slack, volunteers were called for to documents discussion and for giving regular update. Do we have someone interested to do this for the team?
> https://hngi7.slack.com/archives/G0155KA37M2/p1591570646363100?thread_ts=1591570646.363100&cid=G0155KA37M2

Could get in on this

**Comment by acefestus** [comment]
> > Also, the #team-furry page on slack, volunteers were called for to documents discussion and for giving regular update. Do we have someone interested to do this for the team?
> > https://hngi7.slack.com/archives/G0155KA37M2/p1591570646363100?thread_ts=1591570646.363100&cid=G0155KA37M2
> 
> Could get in on this

Please locate that thread on Slack and volunteer for Mobile. Thanks

**Comment by acefestus** [comment]
The covid app can be modified for the following

1. appointment scheduler
2. simple workout reps and sets counter and timer app,
3. running laps counter and timer
4. recipe app and food counter
5. Meeting Reminder
6. Pill reminder
7. Birthday Reminder
8. drink water reminder
9. reading reminder

**Comment by maroafenogho** [comment]
@Maro | @maroafenogho | mobile

**Comment by acefestus** [comment]
@Dera-O please DM me on Slack so we can sort the volunteering thing

**Comment by Trojan9** [comment]
seems our idea was approved by mark

**Comment by Trojan9** [comment]
so lets get going

**Comment by acefestus** [comment]
Okay, we should await response from Frontend right?

**Comment by Trojan9** [comment]
do they already know how the app will work ..like each features?

**Comment by acefestus** [comment]
I have posted that for them

```
We want to design an app for hand-washing reminder for covid19
The primary feature is for people to get a reminder on when to wash their hands
People should be able to keep track of their handwashing activities. Showing and giving kudos for progress
It should also give a global statistics and that of the user's country.
The alert for reminder could be with a certain voice message.
We hope to have secondary features to be added  later but we want the primary features to go out first
```

**Comment by Trojan9** [comment]
@vectaaa are you very good with coding?..we need to start the job

**Comment by Dera-O** [comment]
> @Dera-O please DM me on Slack so we can sort the volunteering thing

Sorted

**Comment by acefestus** [comment]
So another issue was created for planning
Kindly follow there.

@Trojan9 kindly inform the group when a new issue is to be created so people will not get lost.

https://github.com/hng-teamfury-org/mobile/issues/3

**Comment by acefestus** [comment]
Please, if you have been invited to join the Organization's Github, kindly join the mobile project
